### PR TITLE
Practice 5

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

File: Angela/Practice 5 Variables/Exercise1.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Father
+	@@money=50	
+	def self.money
+		@@money
+	end
+end
+
+class Child1 < Father
+	@amount=20
```
Comment (karimeSalomon) [inline Angela/Practice 5 Variables/Exercise1.rb]: Is this code working fine?

File: Angela/Practice 5 Variables/Exercise1.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+class Child3 < Father
+	@amount=15
+	def self.money
+	@@money-=@amount
+	end
+end
+
+puts "Total father's amount = 50"
+puts "Child1: #{Child1.money}"
```
Comment (karimeSalomon) [inline Angela/Practice 5 Variables/Exercise1.rb]: Please add the execution result for this file.

File: Angela/Practice 5 Variables/Exercise4.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+def celciusInFahrenheit(celcius)
+	fahrenheit=(((9*celcius.to_f)/5)+32).round(2)
+end
+
+def fahrenheitInCelcius fahrenheit
+	celcius=((5*(fahrenheit.to_f-32))/9).round(2)
+	return celcius
+end
+
+puts celciusInFahrenheit 35
```
Comment (karimeSalomon) [inline Angela/Practice 5 Variables/Exercise4.rb]: the values should be received by prompt and send with the correct format to the method.

File: Angela/Practice 5 Variables/Exercise1.rb (hunk 1 above)
Comment (angy113) [inline Angela/Practice 5 Variables/Exercise1.rb]: Yes I hesitated just for class variables that wouldn't work, but it worked in that way.

File: Angela/Practice 5 Variables/Exercise4.rb (hunk 3 above)
Comment (angy113) [inline Angela/Practice 5 Variables/Exercise4.rb]: I already corrected it

**Pull request merged**